k8s/demo/common/money-claims/cmc-demo.yaml

```diff
@@ -94,6 +94,7 @@ spec:
             fromSecret: true
             secretKeyRefName: ccd-api-gateway-oauth2-client-secret
             secretKeyRefKey: key
+            CORS_ORIGIN_WHITELIST: https://case-management-web-ccd-cmc-citizen-frontend.demo.platform.hmcts.net
```

Comment:
also shouldn't this be under environment?
pretty sure you can just put in *, but do that later

Reply:
This already seems to be set to this:
```
 CORS_ORIGIN_METHODS:             GET,POST,OPTIONS,PUT,DELETE
 CORS_ORIGIN_WHITELIST:           *
```

Reply:
Yeah ... But I thought perhaps CORS doesn't like the * ... hence I wanted to try being specific.

Reply:
I can fix it, will send PRs on monday I know what to do...

Reply:
Thanks Tim ...